core/src/test/java/org/pentaho/di/core/util/StringEvaluatorTest.java

```diff
@@ -27,6 +27,7 @@
 import java.util.*;
 
 import org.junit.AfterClass;
+import org.junit.Assert;
```

Comment:
Why are you updating this file just to add a new import?
Please revert this file.